applications/GeoMechanicsApplication/tests/test_conditions.py

```diff
@@ -0,0 +1,246 @@
+import os
```

Comment:
Could we add a README.md for these tests (a single one for all of these is probably enough)? That also helps me understand the test a bit better (e.g. which node are we testing at which time-step, which effects are taken into account etc)

Reply:
Thank you @mnabideltares for adding the readme, it helps a lot in understanding the tests!